src/components/Tutorial/TutorialScrollUpButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, useCallback } from 'react'
+import TutorialNavigationButton from './TutorialNavigationButton'
+
+interface TutorialScrollUpButtonProps {
+  show: boolean
+}
```

Comment:
Let's inline this since it's so small.

Reply:
Makes sense. Refactored here: 6c6563bfd3a3c4bd77f942f963a4a14bdccfcdf7